maxim/logger/openai/async_completions.py

```
@@ -1,15 +1,75 @@
-from typing import Optional
+from typing import Any, AsyncIterator, List, Optional
```

Ruff (0.11.9)

Import from `collections.abc` instead: `AsyncIterator`

Import from `collections.abc`

(UP035)